### ISSUE TITLE
Error encountered while running

### ISSUE THREAD

**Issue by cuongnguyen100** [transpect/docx2tex#70]
Here is the error:

> `[ERROR] Unzip: C:\Users\Admin\Desktop\docx2tex\a.docx.tmp\word\media\image12.wmf (Access is denied)
> ERROR: step p:xslt named zip-manifest at file:/C:/Users/Admin/Desktop/docx2tex/docx2hub/xpl/single-tree.xpl:160:docx2hub/xpl/single-tree.xpl:160:58:p:xslt returned non-XML result
> ERROR: step p:xslt named zip-manifest at file:/C:/Users/Admin/Desktop/docx2tex/docx2hub/xpl/single-tree.xpl:160:docx2hub/xpl/single-tree.xpl:160:58:p:xslt returned non-XML result
> ERROR: step p:xslt named zip-manifest at file:/C:/Users/Admin/Desktop/docx2tex/docx2hub/xpl/single-tree.xpl:160:docx2hub/xpl/single-tree.xpl:160:58:p:xslt returned non-XML result
> ERROR: step p:xslt named zip-manifest at file:/C:/Users/Admin/Desktop/docx2tex/docx2hub/xpl/single-tree.xpl:160:docx2hub/xpl/single-tree.xpl:160:58:p:xslt returned non-XML result
> ERROR: p:xslt returned non-XML result
> INFO : xpl/docx2tex.xpl:197:38:No custom-font-maps loaded.
> [ERROR] Unzip: C:\Users\Admin\Desktop\docx2tex\a.docx.tmp\word\media\image12.wmf (Access is denied)
> ERROR: step p:xslt named zip-manifest at file:/C:/Users/Admin/Desktop/docx2tex/docx2hub/xpl/single-tree.xpl:160:docx2hub/xpl/single-tree.xpl:160:58:p:xslt returned non-XML result
> ERROR: step p:xslt named zip-manifest at file:/C:/Users/Admin/Desktop/docx2tex/docx2hub/xpl/single-tree.xpl:160:docx2hub/xpl/single-tree.xpl:160:58:p:xslt returned non-XML result
> ERROR: step p:xslt named zip-manifest at file:/C:/Users/Admin/Desktop/docx2tex/docx2hub/xpl/single-tree.xpl:160:docx2hub/xpl/single-tree.xpl:160:58:p:xslt returned non-XML result
> ERROR: step p:xslt named zip-manifest at file:/C:/Users/Admin/Desktop/docx2tex/docx2hub/xpl/single-tree.xpl:160:docx2hub/xpl/single-tree.xpl:160:58:p:xslt returned non-XML result
> ERROR: p:xslt returned non-XML result`

Here is the file: 
[a.docx](https://github.com/transpect/docx2tex/files/5840218/a.docx)

**Comment by gimsieke** [comment]
Your error message says in its first line: “(Access is denied)”

By default, the docx is unzipped in the directory where it resides. Apparently, you don’t have permissions to create directories (and files) there. The option `extract-dir` lets you specify the file system path where it should be extracted instead. 

Please let us know whether this works.

**Comment by cuongnguyen100** [comment]
> Your error message says in its first line: “(Access is denied)”
> 
> By default, the docx is unzipped in the directory where it resides. Apparently, you don’t have permissions to create directories (and files) there. The option `extract-dir` (given at the end of the command line, that is, where `docx` is also given) lets you specify the file system path where it should be extracted instead.
> 
> Please let us know whether this works.

Sir, can you explain more clearly ?

**Comment by gimsieke** [comment]
Ah, I see, you have probably invoked one of the front-end scripts `d2t` or `d2t.bat`. They don’t seem to have an option to specify a temporary working directory. By default, the directory is used in which the docx file is stored. Maybe @mkraetke can add an option that lets users specify the temporary folder. 

In the meantime, you can copy the docx file into a directory that you have write access to. I wonder how `a.docx` got in that directory in the first place. Are you sure that you have write access to `C:\Users\Admin\Desktop\docx2tex\`?

**Comment by cuongnguyen100** [comment]
I copied that a.docx to docx2tex folder (and the folder is at Desktop). The original directory of the docx file is also at Desktop, sir. Let me try again

**Comment by cuongnguyen100** [comment]
It works perfectly, thank you very much indeed!

**Comment by gimsieke** [comment]
You’re welcome!

**Comment by mkraetke** [comment]
> By default, the directory is used in which the docx file is stored. Maybe @mkraetke can add an option that lets users specify the temporary folder.
As noted in the [README.md](https://github.com/transpect/docx2tex/blob/master/README.md#user-content-run-docx2tex), you can specify an alternate output directory with `-o`.

**Comment by gimsieke** [comment]
Is `-o` available in `d2t.bat`, too?

**Comment by mkraetke** [comment]
There was no similar option but I've added a parameter to pass the output directory.
